framework/src/testing/index.ts

```diff
@@ -13,7 +13,11 @@
  *
  */
 
+import * as fixtures from './fixtures';
```

Comment:
can you directly export instead of importing and then exporting below?

Reply:
In that case, Eslint complains `error  'fixtures' not found in imported namespace 'testing'`

Reply:
ESLint complains of doing `export * as fixtures from './fixtures'`